docs/dataset-overviews/aerosol-optical-properties-macv2-sp.md

```diff
@@ -0,0 +1,49 @@
+# Aerosol optical properties/MACv2-SP
```

Comment:
Should we also include contact info on the data? Linking to data provider info.

Reply:
Yes good point. I'm a little bit hesitant about duplicating information and having it become out of date, but on this way the duplication is the right choice I think.